### PR TITLE
(7.1.0 branch) HL7 parameters and examples

### PR DESCRIPTION
**Pull request by nilminiwso2**
HL7 documentation updated:

- HL7 message store docs removed (because the HL7 console, which was previously available in the management console is no longer available.
- Refactored the parameters, examples, and mediator properties.

### CODE REVIEW

File: en/micro-integrator/docs/references/mediators/property-reference/axis2-properties.md [hunk 1]
```diff
@@ -238,3 +238,300 @@ Axis2 properties allow you to configure the web services engine in WSO2 Micro In
...
+		<td>
+			axis2
+		</td>
+	</tr>
+	<tr>
+		<td>
+			Description
+		</td>
+		<td>
+			Use this property to specify whether an ACK or NACK should be returned to the messaging client as an acknowledgement. If you select a NACK response, you need to specify a custom NACK message by using the <code>HL7_NACK_MESSAGE</code> as given below.
```
Comment (Nirothipan) [inline en/micro-integrator/docs/references/mediators/property-reference/axis2-properties.md:343]:
```suggestion
			Use this property to specify whether an ACK or NACK should be returned to the messaging client as an acknowledgement. If you select a NACK response, you may specify a custom NACK message by using the <code>HL7_NACK_MESSAGE</code> as given below.
```

File: en/micro-integrator/docs/use-cases/examples/hl7-examples/acknowledge_hl7_messages.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
+# Acknowledging HL7 Messages
+
+Automatic message acknowledgement for HL7 messages is enabled in the Micro Integrator by default. With this setting, an ACK is immediately returned to the client when a message is received. 
+
+If required, you can disable automatic acknowledgement. This allows you to control how and when ACK/NACK messages should be returned to the client. Tha is, you can define the integration logic to generate an ack/nack message after message validations or related tasks.
```
Comment (Nirothipan) [inline en/micro-integrator/docs/use-cases/examples/hl7-examples/acknowledge_hl7_messages.md:5]:
```suggestion
If required, you can disable automatic acknowledgement. This allows you to control how and when ACK/NACK messages should be returned to the client. That is, you can define the integration logic to generate an ack/nack message after message validations or related tasks.
```

**Review by Nirothipan — APPROVED** [review]

File: en/micro-integrator/docs/references/mediators/property-reference/axis2-properties.md (hunk 1 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/references/mediators/property-reference/axis2-properties.md:343]: what happens if we don't specify a custom NAK message?

Comment (Nirothipan) [reply en/micro-integrator/docs/references/mediators/property-reference/axis2-properties.md:343]: there won't be a nack message

Comment (nilminiwso2) [reply en/micro-integrator/docs/references/mediators/property-reference/axis2-properties.md]: resolved

File: en/micro-integrator/docs/use-cases/examples/hl7-examples/acknowledge_hl7_messages.md (hunk 2 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/use-cases/examples/hl7-examples/acknowledge_hl7_messages.md]: fixed

**Pull request merged**